spec/models/early_career_payments/slug_sequence_spec.rb

```diff
@@ -0,0 +1,18 @@
+require "rails_helper"
+
+RSpec.describe EarlyCareerPayments::SlugSequence do
+  let(:claim) { build(:claim, eligibility: build(:early_career_payments_eligibility)) }
+
+  subject(:slug_sequence) { EarlyCareerPayments::SlugSequence.new(claim) }
+
+  describe "The sequence as defined by #slugs" do
```

Comment:
Missing specs for other sequences.

Reply:
@nomasprime I have been consistent with what has been delivered in the M+P / TSLR journeys, where rightly or wrongly the spec only covers the exclusions. Happy to write the tests if necessary, but feel it is already covered in the feature spec for the ECP claim journey